types/tinajs__tina-redux/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for @tinajs/tina-redux 1.4
+// Project: https://github.com/tinajs/tina-redux
+// Definitions by: Jiayu Liu <https://github.com/Jimexist>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/tinajs__tina-redux
+// TypeScript Version: 2.3
+
+import { Store, Dispatch } from "redux";
+
+export type MapStateToProps = (state: any) => any;
+export type mapDispatchToProps = (dispatch: Dispatch) => any;
+
+export interface HOC {
+    onLoad: () => void;
+    onUnload: () => void;
+    methods: object;
+}
+
+export default class TinaRedux {
```

Comment:
`node -p 'require("@tinajs/tina-redux")'` is directly a function, not an object with a `default` property. So this should be `class TinaRedux { ... }; export = TinaRedux;`. See the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).